devlab/tests/filtering_utils.py

```diff
@@ -77,3 +80,40 @@ def get_vm_fip(vm):
                     return addr['addr']
         raise RuntimeError('VM with name {} and id {} doesnt have fip'.format(
             vm.name, vm.id))
+
+
+class MigrationUtils(object):
```

Comment:
`MigrationUtils` does not seem to be related to `filtering_utils` module. Maybe renaming `filtering_utils.py` to `utils.py` helps?